src/main/resources/reference.conf

```diff
@@ -2,90 +2,88 @@
 # Kamon-Datadog Reference Configuration #
 # ===================================== #
 
-kamon {
-  datadog {
-
-    #
-    # Settings relevant to the DatadogAgentReporter
-    #
-    agent {
-      # Hostname and port in which your dogstatsd is running (if not using the API). Remember that Datadog packets are sent using UDP and
-      # setting unreachable hosts and/or not open ports wont be warned by the Kamon, your data wont go anywhere.
-      hostname = "127.0.0.1"
-      port = 8125
+kamon.datadog {
+
+  #
+  # Settings relevant to the DatadogAgentReporter
+  #
+  agent {
+    # Hostname and port in which your dogstatsd is running (if not using the API). Remember that Datadog packets are sent using UDP and
+    # setting unreachable hosts and/or not open ports wont be warned by the Kamon, your data wont go anywhere.
+    hostname = "127.0.0.1"
+    port = 8125
+
+    # Max packet size for UDP metrics data sent to Datadog.
+    max-packet-size = 1024 bytes
+    measurement-formatter = "default"
+    packetbuffer = "default"
+  }
 
-      # Max packet size for UDP metrics data sent to Datadog.
-      max-packet-size = 1024 bytes
+  #
+  # Settings relevant to the DatadogSpanReporter
+  #
+  trace {
 
-      measurement-formatter = "default"
+    # Default to agent URL (https://docs.datadoghq.com/api/?lang=python#tracing)
+    api-url = "http://localhost:8126/v0.4/traces"
 
-      packetbuffer = "default"
+    # FQCN of the "kamon.datadog.KamonDataDogTranslator" implementation that will convert Kamon Spans into Datadog
+    # Spans, or "defult" to use the built-in translator.
+    translator = "default"
 
-    }
-
-    #
-    # Settings relevant to the DatadogAPIReporter
+    # HTTP client timeout settings:
+    #   - connect-timeout: how long to wait for an HTTP connection to establish before failing the request.
```

Comment:
These settings changed because as far as I understood, they not necessarily mean what we thought they might mean.